### PR TITLE
Submitting lesson02

### PR DESCRIPTION
**Pull request by mgummel25**

### CODE REVIEW

File: students/mgummel/lesson02/assignment/code/charges_calc.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        3: logging.DEBUG
+    }
+
+    try:
+        # set debug level for which the logger will capture
+        debug_level = level_dict[int(level)]
+        LOGGER.setLevel(debug_level)
+
+        # create file handler which logs even debug messages
+        log_file = logging.FileHandler('charges_calc.log')
```
Comment (ldconejo) [inline students/mgummel/lesson02/assignment/code/charges_calc.py:47]: This one needs to change to reflect the date the log is created.

File: students/mgummel/lesson02/assignment/code/charges_calc.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        log_stdout.setFormatter(formatter)
+        log_file.setFormatter(formatter)
+
+        if debug_level == logging.DEBUG:
+            log_file.setLevel(logging.WARNING)
+
+        LOGGER.addHandler(log_stdout)
+        LOGGER.addHandler(log_file)
+
+    except KeyError:
```
Comment (ldconejo) [inline students/mgummel/lesson02/assignment/code/charges_calc.py:64]: This is a good way to handle an invalid logging level, you should also add a _print_ statement letting the user know they have entered an invalid level.

**Comment by mgummel25** [comment]
Hey Luis,

Thanks for taking a look at this. If I make the changes you pointed out, do
am I able to receive higher points? I have a few assignments I need to
clean up and send it over ASAP over the weekend.

Best,
Matt

**Comment by ldconejo** [comment]
Hi Matt,

Yes, every time I provide any feedback and you submit an updated pull request, I will update your grade based on the latest submission. It's how it works for real software development teams (minus the grades, although getting paid is a bit like getting a grade :-) ), so it's how I do it.

Go ahead and submit any assignments that are ready for review. We still have a few days before your end date. Based on those, we can discuss an extension if you're interested.

Regards,

Luis

**Pull request merged**